tests/test_utils/test_parsing.py

```diff
@@ -1,83 +1,21 @@
 import pytest
 
-from eth_abi.exceptions import (
-    ParseError,
-)
 from eth_abi.utils.parsing import (
     collapse_type,
-    process_type,
-)
-
-
-@pytest.mark.parametrize(
-    'typestr, expected_parse',
-    (
-        ('uint256', ('uint', '256', [])),
-        ('uint', ('uint', '256', [])),
-        ('uint256[]', ('uint', '256', [[]])),
-        ('uint[]', ('uint', '256', [[]])),
-        ('int[2]', ('int', '256', [[2]])),
-        ('function', ('bytes', '24', [])),
-        ('fixed', ('fixed', '128x18', [])),
-        ('ufixed', ('ufixed', '128x18', [])),
```

Comment:
Looks like these last three cases aren't tested by the grammar parser https://github.com/ethereum/eth-abi/blob/master/tests/test_grammar.py#L77

Maybe now is a good time to add them.

Reply:
Added some tests in the `test_grammar` module for those cases.